backend/src/main/scala/com/softwaremill/bootzooka/DependenciesFactory.scala

```diff
@@ -0,0 +1,67 @@
+package com.softwaremill.bootzooka
+
+import cats.data.NonEmptyList
+import cats.effect.{IO, Resource}
+import com.softwaremill.bootzooka.config.Config
+import com.softwaremill.bootzooka.email.{EmailConfig, EmailModel, EmailService}
+import com.softwaremill.bootzooka.email.sender.{DummyEmailSender, EmailSender, MailgunEmailSender, SmtpEmailSender}
+import com.softwaremill.bootzooka.http.{Http, HttpApi, HttpConfig}
+import com.softwaremill.bootzooka.infrastructure.{DB, DBConfig}
+import com.softwaremill.bootzooka.metrics.{MetricsApi, VersionApi}
+import com.softwaremill.bootzooka.passwordreset.{PasswordResetApi, PasswordResetAuthToken, PasswordResetCodeModel}
+import com.softwaremill.bootzooka.security.{ApiKeyAuthToken, ApiKeyModel}
+import com.softwaremill.bootzooka.user.UserApi
+import com.softwaremill.bootzooka.util.{Clock, DefaultIdGenerator, IdGenerator}
+import com.softwaremill.macwire.autocats.autowire
+import doobie.util.transactor.Transactor
+import io.prometheus.client.CollectorRegistry
+import sttp.client3.SttpBackend
+
+object DependenciesFactory {
```

Comment:
I'd prefer to avoid a single class where the dependencies are defined. This won't scale. Let's find a way to dismantle this :)